autobot/repos.py

```diff
@@ -0,0 +1,41 @@
+from issues import check_issue, fetch_issue_info
```

Comment:
```suggestion
from .issues import check_issue, fetch_issue_info
```

ditto for other places